subsys/bluetooth/host/long_work.h

```diff
@@ -0,0 +1,31 @@
+/* long_work.h - Workqueue intended for long-running operations. */
+
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#if defined(CONFIG_BT_LONG_WQ)
+
+void bt_long_work_schedule(struct k_work_delayable *dwork, k_timeout_t timeout);
```

Comment:
Seems like you forgot to update the return types here?

Reply:
yep good catch. fixed